middle_end/flambda2/simplify/non_constructed_code.mli

```diff
@@ -19,4 +19,55 @@
 (* Code without any function bodies, but with all the associated metadata, e.g.
    free names. *)
 
-include Code_intf.S with type function_params_and_body := unit
+type t = unit Code0.t
```

Comment:
At one point we're going to have to start adding doc comments in these mli files...

Reply:
I've changed this one to a doc comment.